subsys/bluetooth/audio/endpoint.h

```diff
@@ -134,3 +138,8 @@ static inline const char *bt_audio_ep_state_str(uint8_t state)
 bool bt_audio_ep_is_broadcast_snk(const struct bt_audio_ep *ep);
 bool bt_audio_ep_is_broadcast_src(const struct bt_audio_ep *ep);
 bool bt_audio_ep_is_unicast_client(const struct bt_audio_ep *ep);
+
+typedef void (*bt_audio_ase_func_t)(const struct bt_audio_ep_info *info,
+		void *user_data);
+void bt_unicast_server_foreach_ase(struct bt_conn *conn,
+		bt_audio_ase_func_t func, void *user_data);
```

Comment:
These and the `struct bt_audio_ep_info` should be put into the public API in https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/bluetooth/audio/audio.h so that it's usable by applications